index.html

```diff
@@ -365,14 +661,38 @@ <h4>
           short noindices>
         </ndef-record>
       </ndef-record>
+    </div>
+    </section>
+
+    <section> <h5>Signature records</h5>
+    <p>
+      <dfn>NDEF Signature</dfn> is defined [[NDEF-SIGNATURE]].
+      Its <a>TYPE field</a> contains "`Sig`" (`0x53`, `0x69`, `0x67`) and its
+      <a>PAYLOAD field</a> contains version, signature and a certificate chain.
+      Web NFC does not support this at the moment.
     </p>
     </section>
+
+    <section> <h5>Handover records</h5>
+    <p>
+      <dfn>NFC handover</dfn> is defined [[NFC-HANDOVER]] and the
+      corresponding message structure that allows negotiation and activation of
+      an alternative communication carrier, such as Bluetooth or WiFi.
+      The negotiated communication carrier would then be used (separately) to
+      perform certain activities between the two devices, such as sending photos
+      to the other device, printing to a Bluetooth printer or streaming video to
+      a television set. Web NFC does not support this at the moment.
```

Comment:
I rather we add an issue and create a "note" or "issue" or what it is called in respec, linking to that

Reply:
IMV it's orthogonal if we have or don't have an issue for the missing feature. But we could do it (later), it would be nice.

Reply:
Added #364, see also #313

Reply:
Added a comment and reference to the issue.